### ISSUE TITLE
Question: How to use datetime type with SelfQueryRetriever

### ISSUE THREAD

**Issue by wsantos** [langchain-ai/langchain#4801]
### Issue you'd like to raise.

How can one use SelfQueryRetriever to query `datetime`, I've added a `datetime` object as string to the metadata, not sure if that is right or if we should use `timestamp`, but things got weird with this prompt:

```
I want to watch a movie rated higher than 8.5 and released today
```

```json
{
    "query": "",
    "filter": "and(gt(\"rating\", 8.5), eq(\"released\", \"today\"))"
}
```

I'm not sure how to instruct langchain to convert today to a datetime/str.

Is there a from/to AttributeInfo so we can convert when saving and loading from the vectorstore?

Long story short, how would you guys approach this scenario?

Thanks in advance.

**Comment by MehrCurry** [comment]
@wsantos How did you solve that issue? I have the same problem. I am trying to convert the timestamp to unixtime as an int64 but had limited success with it.

**Comment by wsantos** [comment]
@MehrCurry I did not solve it, sorry.
